tests/OWNERS

```diff
@@ -1,5 +1,4 @@
 # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
 
 approvers:
-- amitkrout
 - mohammedzee1000
```

Comment:
we should not end up with just a review.
Can we include @prietyc123 as approver and @rnapoles-rh as a reviewer?

Reply:
done